examples/VNP09H1/VNP09H1.A2012017.h00v09.001.2016294114238.json

```
@@ -0,0 +1,554 @@
+{
+  "type": "Feature",
+  "stac_version": "1.0.0",
+  "id": "VNP09H1.A2012017.h00v09.001.2016294114238",
+  "properties": {
+    "start_datetime": "2012-01-17T00:00:00Z",
+    "end_datetime": "2012-01-25T00:00:00Z",
+    "viirs:horizontal-tile": 0,
+    "viirs:vertical-tile": 9,
+    "viirs:tile-id": "51000009",
+    "created": "2016-10-20T11:42:38Z",
+    "platform": "Suomi National Polar-Orbiting Partnership (SNPP)",
```

`platform` is generally `lower-snake-case`, reference: https://github.com/radiantearth/stac-spec/blob/master/item-spec/common-metadata.md#platform.
```suggestion
    "platform": "snpp",
```

---

yep, good catch. Fixed.